layouts/shortcodes/iframe.html

```diff
@@ -0,0 +1,5 @@
+{{ $url := .Get "url" }}
+<div style="width: 100%; height:calc(100vh - 200px)">
+    <iframe src="{{ $url }}" height="100%" width="100%"><a href="{{ $url }}">API Documentation</a></iframe>
```

Comment:
Keep it simple. Introducing something just in case is almost always an indicator of unnecessary complexity. It's important to get comfortable with the possibility that this may need to be adjusted, and it's easier to do so when it's simple.